crates/nu-std/tests/test_formats.nu

```diff
@@ -0,0 +1,54 @@
+use std assert
+
+def ndjson_test_data1 [] {
+  '{"a":1}
+{"a":2}
+{"a":3}
+{"a":4}
+{"a":5}
+{"a":6}'
+}
+
+#[test]
+def from_ndjson_multiple_objects [] {
+  use std formats *
+  let result = ndjson_test_data1 | from ndjson
+  let expect = [{a:1},{a:2},{a:3},{a:4},{a:5},{a:6}]
+  assert equal $result $expect "from ndjson successfully"
```

Comment:
because `"from ndjson successfully"` would be the failing message, it might make more sense to say `"could not convert to NDJSON"`?

same goes with the other `assert` messages :yum:

Reply:
close enough: `from ndjson` ;)